include/eve/arch/spec.hpp

```diff
@@ -8,10 +8,11 @@
 #pragma once
 
 #if !defined(EVE_NO_SIMD)
-#  include <eve/arch/x86/spec.hpp>
-#  include <eve/arch/ppc/spec.hpp>
-#  include <eve/arch/arm/sve/spec.hpp>
 #  include <eve/arch/arm/neon/spec.hpp>
+#  include <eve/arch/arm/sve/spec.hpp>
```

Comment:
Again, there is no need to reorder those

Reply:
Accidental change by clang-format. Fixed.